processor/otel/consumer.go

```diff
@@ -408,6 +418,71 @@ func parseSpan(span *tracepb.Span, event *model_span.Event) {
 	event.Labels = labels
 }
 
+func parseErrors(source string, otelSpan *tracepb.Span) []*model_error.Event {
+	var errors []*model_error.Event
+	for _, log := range otelSpan.GetTimeEvents().GetTimeEvent() {
+		var isError bool
+		var logMessage, exMessage string
+		for k, v := range log.GetAnnotation().GetAttributes().GetAttributeMap() {
+			if source == sourceFormatJaeger {
+				switch v := v.Value.(type) {
+				case *tracepb.AttributeValue_StringValue:
+					vStr := v.StringValue.Value
+					switch k {
```

Comment:
what about "message"?
https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table

Reply:
Maybe also "error.kind" for exception.type?

Reply:
I handled all the fields that were set in the examples I saw. But I will give it another thought of how to make this more comprehensive.

Reply:
This is good enough for me now, I think we could extend it later.